pymc/gp/cov.py

```diff
@@ -812,6 +824,52 @@ def full(self, X, Xs=None):
     def diag(self, X):
         X, _ = self._slice(X, None)
         return self.cov_func(self.w(X, self.args), diag=True)
+    
+
+class WrappedPeriodic(Covariance):
```

Comment:
I think you had GeneralizedPeriodic originally as the name, why the switch?  I think GeneralizedPeriodic makes it a bit clearer what it's doing.

Reply:
I felt it captured better what it was doing i.e. you use it to wrap up an existing kernel to make it periodic. I think a good name might be a verb (like `Add` or `Prod`) since it acts on an existing kernel...but I don't know what that verb would be :) `Periodify`...

Reply:
Makes sense.  I guess `Wrapped` is more describes what the code does, and `Generalized` describes what the kernel is.  Either way makes sense.